modules/database/src/ioc/db/dbChannelIO.cpp

```diff
@@ -165,7 +165,7 @@ unsigned long dbChannelIO::nativeElementCount (
 {
     guard.assertIdenticalMutex ( this->mutex );
     long elements = dbChannelElements ( this->dbch );
-    if ( elements >= 0u ) {
+    if ( elements >= 0 ) {
```

Comment:
`dbChannelElements()` does evaluate to a signed type, so comparing to signed zero is the strictly correct action.